README.md

```diff
@@ -0,0 +1,3 @@
+## Happy Website Readme
+
+This is an happy Website
```

Comment:
you should add the name of the collaborators please

Reply:
yes, sorry, you are right

Reply:
will do immediately